### PR TITLE
stats bar

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: about-us.php [hunk 1]
```diff
@@ -1,4 +1,41 @@
 <?php include 'includes/header.php';?>
 
+<div class="hero-image full-width bottom-spacer">
```
Comment (ElXyfer) [inline about-us.php]: Hero image code should not be in this pull request.

File: learning/tutorial-five.php [hunk 2]
```diff
@@ -32,14 +32,12 @@
 				$('#result').html(data);
 			});
 			*/
-			/*
 			//gets JSON encoded data 
 			$.getJSON('users.json', function(data){
 				$.each(data, function(i, user){
-					$('ul#users').append('<li>'+user.firstName+'</li>');
+					$('ul#users').append('<li class="'+user.lastName+'">'+user.firstName+'</li>');
```
Comment (ElXyfer) [inline learning/tutorial-five.php]: this code is not part of the status bar.

File: theme/js/script.js [hunk 3]
```diff
@@ -118,3 +118,11 @@ function addActiveClassToNavbar() {
         }
     });
 };
+
+function getJsonData() {
```
Comment (ElXyfer) [inline theme/js/script.js]: this code is not part of the status bar.

File: about-us.php (hunk 1 above)
Comment (katlazarova) [reply about-us.php]: done

File: learning/tutorial-five.php (hunk 2 above)
Comment (katlazarova) [reply learning/tutorial-five.php]: okay done

File: theme/js/script.js (hunk 3 above)
Comment (katlazarova) [reply theme/js/script.js]: I don't see this code in my current branch

**Pull request merged**